docs/src/threading.rst

```diff
@@ -59,6 +59,38 @@ Threads
 
     .. versionchanged:: 1.4.1 returns a UV_E* error code on failure
 
+.. c:function:: int uv_thread_setaffinity(uv_thread_t* tid, char* cpumask, char* oldmask, size_t mask_size)
+
+    Sets the specified thread's affinity to cpumask, which is specified in
```

Comment:
Maybe add a brief explanation on the `mask_size` argument to both affinity functions.